fix: client credential cache (#626)
SAP/cloud-sdk: CHANGELOG.md

## Fixed Issues

- [Generator] Fix the function import (OData V2 + V4) and action import, where the return type is a primitive edm type like `Edm.String`.

> - [Cache] Fix the client credential cache, which is never hit due to wrong expire time.

Reviewer: I think it is `client credentials` with an "s"

> - [Cache] Fix the client credentials cache, which is never hit due to wrong expire time.

Reviewer 2: ```suggestion
- [Cache] Fix expiration times in the client credentials cache.
```
